app/src/main/AndroidManifest.xml

```diff
@@ -35,6 +35,12 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="${applicationId}"  />
+            </intent-filter>
```

Comment:
This will have the app handle the scheme matching the applicationId. 

i.e. if the application id is "org.openedx.app" then this will handle: com.openedx.app://